dao-contracts/src/dao_nft.rs

```diff
@@ -0,0 +1,89 @@
+use casper_dao_erc721::{
+    core::ERC721Token, BurnableERC721, MetadataERC721, MintableERC721, TokenId, TokenUri,
+};
+use casper_dao_modules::AccessControl;
+use casper_dao_utils::{
+    casper_dao_macros::{casper_contract_interface, Instance},
+    Address, Mapping,
+};
+use casper_types::U256;
+use delegate::delegate;
+
+#[casper_contract_interface]
+pub trait DaoOwnedNftContractInterface {
+    fn init(&mut self, name: String, symbol: String, base_uri: TokenUri);
+    fn change_ownership(&mut self, owner: Address);
+    fn add_to_whitelist(&mut self, address: Address);
+    fn remove_from_whitelist(&mut self, address: Address);
+    fn get_owner(&self) -> Option<Address>;
+    fn is_whitelisted(&self, address: Address) -> bool;
+    fn name(&self) -> String;
+    fn symbol(&self) -> String;
+    fn owner_of(&self, token_id: TokenId) -> Address;
```

Comment:
@zie1ony should we standardize the return type to be an Option instead of reverting for those kinds of getters?

Reply:
Good idea.

Reply:
Here I would prefer to revert cause there is more than one reason for failure. It could be either an invalid token id or a non-existent owner. It's not only a getter but the method is utilized by some other methods like `approve()`. And the oracle - open zepplin does the same thing. :wink: